gms_app/tests.py

```diff
@@ -3,7 +3,7 @@
 import pytest
 from django.db.utils import IntegrityError
 
-from gms_app import models
+from gms_app.models import Garden, GardenBed, Plant, Planting
```

Comment:
This is a nit of mine, especially in test modules, calling out the models to be tested explicitly in the import